docs/hugo/content/design/_index.md

```diff
@@ -27,6 +27,10 @@ For background information, check out [this Cognitect blog entry](https://www.co
 
 ADR documents should be updated over time to keep them relevant, typically by updating the *Experience Report* section.
 
+## 2024
+
+[**Upstream Deletion**]({{< relref "ADR-2024-02-Upstream-Deletion" >}}) - when might deletion of an upstream resource occur and how will the ASO project handle it.
```

Comment:
minor, phrasing: "The ASO project" reads slightly funky to me.

Maybe just "how will ASO handle it"?

Reply:
Fixed.